images/nginx/default.conf

```diff
@@ -0,0 +1,19 @@
+server {
+    listen	80;
+
+	server_name	phpshell.localhost;
+	root		/srv/http/phpshell.localhost/phpshell/htdocs;
+
+    location ~* \.(css|js|gif|jpe?g|png|ico)$ {
```

Comment:
Maybe there is a better approach for this nginx conf.

(Looks like I mixed tabs and spaces)